assets/app/views/builds.html

```diff
@@ -143,62 +141,77 @@
-              <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
-              <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-              <span ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true"></span>
-            </span>
-          </div>
-          <div>
-            <span style="margin-right: 5px;">Duration: </span>
-            <span ng-switch="build.status.phase" class="hide-ng-leave">
-              <span ng-switch-when="Complete">{{(build.status.startTimestamp || build.metadata.creationTimestamp) | duration : build.status.completionTimestamp}}</span>
-              <span ng-switch-when="Failed">{{build.status.startTimestamp | duration : build.status.completionTimestamp}}</span>
-              <span ng-switch-when="Running">running for <duration-until-now timestamp="build.status.startTimestamp"></duration-until-now></span>
-              <span ng-switch-when="New">waiting for <duration-until-now timestamp="build.metadata.creationTimestamp"></duration-until-now></span>
-              <span ng-switch-when="Pending">waiting for <duration-until-now timestamp="build.metadata.creationTimestamp"></duration-until-now></span>
-              <span ng-switch-default>
-                <span ng-if="build.status.startTimestamp">{{build.status.startTimestamp | duration : build.status.completionTimestamp}}</span>
-                <span ng-if="!build.status.startTimestamp">waited for {{build.metadata.creationTimestamp | duration : build.status.completionTimestamp}}</span>
-              </span>
-            </span>
-          </div>
-          <div ng-if="buildConfig.spec.strategy.type != build.spec.strategy.type">
-            <div>Build strategy: {{build.spec.strategy.type}}</div>
-          </div>
-          <div ng-switch="build.spec.strategy.type">
-            <div ng-switch-when="Source" ng-if="build.spec.strategy.sourceStrategy.from">
-              Builder image: {{build.spec.strategy.sourceStrategy.from | imageObjectRef : build.metadata.namespace}}
+        <div class="well build-unit" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
+          <div class="row">
+            <div class="col-md-7 col-sm-7 col-xs-7">
+              <h3>{{build.metadata.name}}</h3>
             </div>
-            <div ng-switch-when="Docker">
-              <div ng-if="build.spec.strategy.dockerStrategy.from">
-                Builder image: {{build.spec.strategy.dockerStrategy.from | imageObjectRef : build.metadata.namespace}}
+            <div class="col-md-4 col-sm-4 col-xs-offset-1 col-xs-4 text-right">
+              <div>
+                <button class="btn btn-default" ng-click="cloneBuild(build.metadata.name)" ng-disabled="(buildConfigBuildsInProgress[buildConfigName] | hashSize) > 0">Rebuild</button>
               </div>
```

Comment:
Moved "Rebuild" button to be aligned with build name - upper right

https://raw.githubusercontent.com/sg00dwin/origin/browse-view-tile-2column/docs/screenshots/browse-builds.png